features/base/test/test_kernel_config.py

```diff
@@ -8,3 +9,72 @@ def test_nvme_kernel_parameter(client, aws):
     (exit_code, output, error) = client.execute_command("grep -c nvme_core.io_timeout=4294967295 /proc/cmdline")
     assert exit_code == 0, f"no {error=} expected"
     assert output.rstrip() == "1", "Expected 'nvme_core.io_timeout=4294967295' kernel parameter"
+
+
+@pytest.mark.security_id(963)
+def test_for_restricted_access_to_dmesg(client):
+    dmesg_restrict = check_for_kernel_setting(client, "kernel.dmesg_restrict")
+    assert dmesg_restrict == "1", "dmesg wasn't restircted!"
```

Comment:
```suggestion
    assert dmesg_restrict == "1", "dmesg wasn't restricted!"
```

please use a spell checker